yi-core/yi-core.cabal

```diff
@@ -135,3 +136,54 @@ library
       System.CanonicalizePath
       Yi.Buffer.Implementation
       Paths_yi_core
+
+test-suite tasty
```

Comment:
I don't understand how you got this in your Cabal file. Whenever I run hpack, it does not add in the dependencies like this and I get a minimal, 1-1 correspondence with `package.yaml`